src/app/screens/Liquid/ConfirmSignPset.tsx

```diff
@@ -0,0 +1,232 @@
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import PublisherCard from "@components/PublisherCard";
+import SuccessMessage from "@components/SuccessMessage";
+import { TFunction } from "i18next";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import Hyperlink from "~/app/components/Hyperlink";
+import Loading from "~/app/components/Loading";
+import ScreenHeader from "~/app/components/ScreenHeader";
+import { useNavigationState } from "~/app/hooks/useNavigationState";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import api from "~/common/lib/api";
+import { Esplora, EsploraAssetInfos } from "~/common/lib/esplora";
+import msg from "~/common/lib/msg";
+import {
+  Address,
+  PsetPreview,
+  fetchAssetRegistry,
+  getPsetPreview,
+} from "~/common/lib/pset";
+import type { OriginData } from "~/types";
+
+function toLiquidNetworkName(net: "bitcoin" | "testnet" | "regtest") {
+  return net === "bitcoin" ? "liquid" : "testnet"; // TODO: how do we handle regtest ?
```

Comment:
this can be updated now 🎉

Reply:
Edit: if "regtest" is unsupported maybe it would be better to throw an exception here?

Reply:
@louisinger could you please review this one?

Reply:
@rolznz do u mean to support regtest ? If yes, it's fine to return "regtest" here: everything else should be unchanged (it's just a matter of address encoding)

Reply:
Fixed in https://github.com/getAlby/lightning-browser-extension/pull/2583